cli/tests/setup_suite.bash

```diff
@@ -355,6 +355,9 @@ pkgdb_vars_setup() {
   PKGDB_NIXPKGS_REF_OLD="github:NixOS/nixpkgs/$PKGDB_NIXPKGS_REV_OLD"
   PKGDB_NIXPKGS_REF_NEW="github:NixOS/nixpkgs/$PKGDB_NIXPKGS_REV_NEW"
 
+  PKGDB_NIXPKGS_NAR_HASH_OLD="sha256-1UGacsv5coICyvAzwuq89v9NsS00Lo8sz22cDHwhnn8="
+  PKGDB_NIXPKGS_NAR_HASH_NEW="sha256-5uA6jKckTf+DCbVBNKsmT5pUT/7Apt5tNdpcbLnPzFI="
```

Comment:
question: should we generate them dynamically?

seems easy to miss to change when we change the revisions at some point.
No strong opinion.

```suggestion
PKGDB_NIXPKGS_NAR_HASH_OLD="$(nix flake prefetch "$PKGDB_NIXPKGS_REF_OLD" | jq .hash)"
PKGDB_NIXPKGS_NAR_HASH_NEW="$(nix flake prefetch "$PKGDB_NIXPKGS_REF_NEW" | jq .hash)"

```

Reply:
Since they're right next to each other I think it will be obvious enough, especially since they'll trigger test failures